data/actionsmap/yunohost.yml

```diff
@@ -1384,3 +1384,28 @@ hook:
                 -d:
                     full: --chdir
                     help: The directory from where the script will be executed
+
+#############################
+#          Journals         #
+#############################
+journals:
+    category_help: Manage debug journals
+    actions:
+
+        ### journals_list()
+        list:
+            action_help: List journals
+            api: GET /journals
+            arguments:
+               -l:
+                    full: --limit
+                    help: Maximum number of journals per categories
```

Comment:
Journals are arranged in categories : right now there is only the "app" category iirc (it's been a while since I've made this PR).